app/api/routes.py

```diff
@@ -199,11 +200,10 @@ def get_categories():
         ]
 
     except Exception as e:
-        print_tb(e.__traceback__)
-        print(e)
-        category_list = []
-    finally:
-        return standardize_response(payload=dict(data=category_list))
+        logger.error(e)
```

Comment:
I prefer `logger.exception(e)`

Reply:
I didn't know `logger.exception()` was a method. I'm used to the syslog levels of logging (Alert, Critical, Error, Warning, Notice, Informational, Debug) and the related methods associated (i.e., `error()`, `warn()`, `debug()`). I'll have to read up on logger.exception to see what the differences are.

Reply:
Yeah it uses the error level for syslog handling but it also gives some more exception specific details.